### PR TITLE
MBS-10074: '-' on release search format/tracks if no medium exists

### PR DESCRIPTION
**Pull request by reosarevok**
https://tickets.metabrainz.org/browse/MBS-10074

### CODE REVIEW

File: root/search/components/ReleaseResults.js [hunk 1]
```diff
@@ -37,8 +37,16 @@ function buildResult(result, index) {
...
-      <td>{release.combined_track_count}</td>
+      <td>
+        {release.combined_format_name
+          ? release.combined_format_name
+          : '-'}
+      </td>
+      <td>
+        {release.combined_track_count
+          ? release.combined_track_count
+          : '-'}
```
Comment (mwiencek) [inline root/search/components/ReleaseResults.js]: (Nitpick: `{release.combined_track_count || '-'}` is shorter and does the same thing.)

**Comment by reosarevok** [comment]
@yvanzo suggested using "n/a" on both lists (because it conveys meaning much more clearly). What do you think about that alternative?

**Comment by mwiencek** [comment]
Either works for me, though I guess people might wonder why it's not applicable. `[no media]` or something might be more to the point if we want to be descriptive? (Also, I wonder if we're not being consistent with brackets, since we show `(unknown)` for unset formats where elsewhere we use `[unknown]`, `[removed]`, `[none]`, `[missing editor]`, ...)

**Comment by reosarevok** [comment]
What about having [missing media] on the format column, but stick to - in the track number one? Since both will always be displayed together AFAICT, and I feel two [missing media] columns by each other might be a bit much. But maybe I'm overthinking it.

**Comment by mwiencek** [comment]
> What about having [missing media] on the format column, but stick to - in the track number one?

Sounds good to me

**Pull request merged**